src/screens/DiscoveryScreenV2/index.js

```diff
@@ -344,7 +340,7 @@ const DiscoveryScreenV2 = ({route}) => {
           hideBackIcon
         />
       )}
-      {renderFragment()}
+      {RenderFragment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `RenderFragment` instead of invoking `renderFragment()` is correct since `RenderFragment` is now a memoized component.
```diff
-      {renderFragment()}
+      {RenderFragment}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->